server/model/group.js

```diff
@@ -9,12 +9,12 @@ class Group extends BeanModel {
      * @param {boolean} [showTags=false] Should the JSON include monitor tags
      * @returns {Object}
```

Comment:
Can you update the JSDoc comment to reflect the addition of the new parameter
```suggestion
     * @param {boolean} [showStatus = false] Should the JSON include the status
     * @returns {Object}
```